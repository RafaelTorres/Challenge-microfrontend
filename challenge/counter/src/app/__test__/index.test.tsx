import HomePage from '@root/src/pages';
import { render, screen } from '@testing-library/react';

const renderPage = () => render(<HomePage />);

describe('Root Page', () => {
  beforeEach(() => {
    jest.clearAllMocks();
  });

  it('should render home page', () => {
    const { container } = renderPage();
    const button = screen.queryByText(/Counter MF/);
    expect(button).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
