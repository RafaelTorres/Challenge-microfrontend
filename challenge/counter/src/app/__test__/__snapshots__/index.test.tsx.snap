// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root Page should render home page 1`] = `
<div>
  <main>
    <h2>
      Counter MF
    </h2>
    <p
      class="counter-label"
    >
      0
    </p>
  </main>
</div>
`;
